lib/screen/restaurant_menu_screen.dart

```diff
@@ -55,27 +58,69 @@ class RestaurantMenuScreen extends StatelessWidget {
               isBold: true,
               padding: const EdgeInsets.symmetric(vertical: 15),
             ),
-
             TextTitleSmall(
               text: restaurantDescription,
               padding: EdgeInsets.zero,
             ),
-
             const SectionHeaderTB(text: 'Uncategorized'),
             ..._getContent(),
-            const SizedBox(height: 16),
-            Align(
-              alignment: Alignment.center,
-              child: ElevatedButton(
-                onPressed: () {
-                  // Complete order functionality
-                },
-                child: const Text('Complete Order'),
-              ),
-            ),
+            const SizedBox(height: 90.0),
           ],
         ),
       ),
+      bottomSheet: Padding(
+        padding: const EdgeInsets.symmetric(vertical: 8.0, horizontal: 15.0),
+        child: SizedBox(
+          height: 60.0,
+          child: Row(
+            children: [
+              Expanded(
+                  flex: 5,
+                  child: GestureDetector(
+                    onTap: () {
+                      Navigator.of(context).push(
+                        PageRouteBuilder(
+                          pageBuilder:
+                              (context, animation, secondaryAnimation) =>
+                                  const ShoppingCart(),
+                          transitionDuration: Duration.zero,
+                        ),
+                      );
+                    },
+                    child: Container(
+                        height: 50.0,
+                        decoration: BoxDecoration(
+                            border: Border.all(color: Constants.primaryColor),
+                            borderRadius: BorderRadius.circular(8.0),
+                            color: Colors.transparent),
+                        child: const Center(
+                          child: TextTitleMedium(
+                            text: "Shopping cart",
+                            color: Constants.primaryColor,
+                          ),
+                        )),
+                  )),
+              const SizedBox(
+                width: 15.0,
+              ),
+              Expanded(
```

Comment:
same here, Use button.dart instead.  all repeated components should better define a new component class.